webapp/config/models.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from django.db import models
+
+
+class ModelManager(models.Manager):
+    use_for_related_fields = True
+
+    def get_queryset(self):
+        return super().get_queryset()
+
+
+class DeletedModelManager(ModelManager):
+    use_for_related_fields = True
+
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted_at__is_null=True)
+
+
+class BaseModel(models.Model):
+    class Meta:
+        abstract = True
+
+    objects = ModelManager()
+
+    created_at = models.DateTimeField(
+        verbose_name='추가된 일시',
+        auto_now_add=True
+    )
+    updated_at = models.DateTimeField(
+        verbose_name='수정된 일시',
+        auto_now=True
+    )
+
+
+class BaseModelWithDeleted(BaseModel):
+    class Meta:
+        abstract = True
+
+    objects = DeletedModelManager()
```

Comment:
```suggestion
    objects = SoftDeleteBaseModelManager()
```
위 리뷰대로 하면, 모델명 뒤에 Manager를 붙여서 일관성있게 해주세욥